tests/run/specialfloat.pyx

```diff
@@ -167,3 +167,36 @@ def global_floats():
     nan
     """
     return (cdef_float_nan, cdef_float_infp, cdef_float_infn)
+
+
+def truthy_float(x: float):
```

Comment:
Would `float | None` work for what you're really trying to test here?

Reply:
Good idea. Done.